indico/modules/rb/schemas.py

```diff
@@ -251,6 +251,12 @@ def _get_permissions(self, booking):
             admin_permissions = {x: getattr(booking, x)(session.user) for x in methods}
         return {'user': user_permissions, 'admin': admin_permissions}
 
+    @post_dump
+    def _hide_sensitive_data(self, data, **kwargs):
+        if not self.context.get('include_notes'):
```

Comment:
You can do the check directly here w/o passing the context. Alternatively you could also use `ReservationDetailsSchema(exclude=['notes'])` to avoid the `post_dump` altogether

```suggestion
    @post_dump(pass_original=True)
    def _hide_sensitive_data(self, data, booking, **kwargs):
        if not booking.room.can_manage(session.user):
```

Reply:
The context was my suggestion because I don't really like the idea of accessing the session etc in a schema. But later I noticed that we do so for some other things as well, so maybe it's not THAT ugly to do it in this particular schema...

Reply:
Yeah, this seems like a much nicer way to handle perms - added :)